csharp/ql/src/semmle/code/csharp/PrintAst.qll

```diff
@@ -48,6 +48,8 @@ private predicate isNotNeeded(Element e) {
   or
   e instanceof TupleType
   or
+  e instanceof ConditionalDirective
```

Comment:
Why not include all preprocessor directives?

Reply:
We can do that, but only `ConditionalDirective`s have expressions underneath them.